services/command-router/src/main/java/org/eclipse/hono/commandrouter/app/DeviceConnectionInfoProducer.java

```diff
@@ -94,6 +93,9 @@ BasicCache<String, String> cache(
                     infinispanCacheConfig,
                     commonCacheConfig);
         }
+         */
+        LOG.info("Creating a new REDIS cache.");
+        return new RedisCache<String, String>();
```

Comment:
Hard coding for now so ignore these changes. Eventually, the type of cache created will depend on the configuration as it was previously (with the addition of a redis config block).